### DISCUSSION TITLE
Need help to customize topbar

### DISCUSSION THREAD

**Discussion by woodsjr007** [end-4/dots-hyprland#707] [Q&A]
![swappy-20240726_095510](https://github.com/user-attachments/assets/fd83b2ae-d427-4b19-9c1a-85feaa656e90)

Hello,
I am new to this dots and not totaly aware of codebase. Can someone help me what changes I need to do for following?

As shown in above screenshot I want to reduce workspace from 10 to 5 on topbar. Also I want to remove those cpu stats widget.
And on left where it shows which app is running, I want to remove subtext which shows long description.

Please point me out where do I need to do this changes in code. Thank you.

**Comment by end-4** [comment]
i'm not going to just give you code and exact lines since i can't be bothered to do this for literally everyone
but you wanna mess with the `~/.config/ags/modules/bar/normal` folder
